### PR TITLE
Fixed meaning of nproc

### LINKED ISSUE #32694: [enterprise-4.6] Issue in file nodes/containers/nodes-containers-using.adoc
### Which section(s) is the issue in?

"where nproc is the number of process reported by the nproc command. "

### What needs fixing?
"process" is wrong, correct  would be "processors"/"CPUs" or rather "processing units" as given by "man nproc".

### PR DESCRIPTION
**Pull request by ehvs**
Resolves #32694

- Aligned team: 
- For branches: 4.5 - 4.6 - 4.7 - 4.8
- https://github.com/openshift/openshift-docs/issues/32694
- Direct link to doc preview: https://deploy-preview-32776--osdocs.netlify.app/openshift-enterprise/latest/nodes/containers/nodes-containers-using.html
- QE review: Waiting for review
- Peer review: Waiting for review 
Please merge @Preeticp

### CODE REVIEW

**Comment by rphillips** [comment]
lgtm

**Comment by haircommander** [comment]
LGTM

File: nodes/containers/nodes-containers-using.adoc [hunk 1]
```diff
@@ -28,11 +28,11 @@ multi-host installations.
...
+The amount of memory stored in those caches is proportional to the number of CPUs that the system uses. As a result, a higher number of CPUs results in a greater amount of kernel memory being held in these caches. Higher amounts of kernel memory in these caches can cause {product-title} containers to exceed the configured memory limits, resulting in the container being killed.
 
-To avoid losing containers due to kernel memory issues, ensure that the containers request sufficient memory. You can use the following formula to estimate the amount of memory consumed by the `kmem_cache`, where `nproc` is the number of process reported by the `nproc` command. The lower limit of container requests should be this value plus the container memory requirements:
+To avoid losing containers due to kernel memory issues, ensure that the containers request sufficient memory. You can use the following formula to estimate the amount of memory consumed by the `kmem_cache`, where `nproc` is the number of processing units available reported by the `nproc` command. The lower limit of container requests should be this value plus the container memory requirements:
```
Comment (lbarbeevargas) [inline nodes/containers/nodes-containers-using.adoc:35]: Small wording suggestion for consideration:
s/the number of processing units available reported by/the number of processing units available that are reported by/

**Review by lbarbeevargas — COMMENTED** [review]
Really small wording suggestion, but otherwise LGTM.

**Review by giuseppe — APPROVED** [review]

**Comment by ehvs** [comment]
Thank you for your suggestions @lbarbeevargas , Ive applied them!
@giuseppe  lmk if it is good to merge

**Comment by giuseppe** [comment]
>  @giuseppe lmk if it is good to merge

yes, LGTM

**Pull request merged**